tests/unit/fixtures/auth.js

```diff
@@ -2,4 +2,7 @@ export const code = '111111';
 
 export const email = 'user@email.com';
 
+// ATTENTION: don't forget to update anotherPasswordWithEqualLength
 export const regularPassword = '11111111';
+
+export const anotherPasswordWithEqualLength = '22222222';
```

Comment:
Yes, because it's very important for tests. If we don't add this info, we will be able to change password with diff length and have problem with tests